src/bin/inx-chronicle/api/stardust/poi/routes.rs

```diff
@@ -91,11 +97,15 @@ async fn create_block_referenced_proof(database: Extension<MongoDb>, Path(block_
         .into());
     }
 
-    Ok(CreateProofResponse {
-        milestone: milestone.into(),
-        block: block.into(),
-        audit_path: merkle_audit_path.into(),
-    })
+    if matches!(headers.get(ACCEPT), Some(header) if header == BYTE_CONTENT_HEADER) {
+        todo!("return proof in binary format");
```

Comment:
Just a reminder comment

Reply:
thanks, but after some consideration the idea of returning a binary response has been dropped for now until there's a use-case for it. The JSON response can easily be converted to whatever the user wants/requires client-side. If there's need for compression then this should be done client-side IMO.